CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Grafana Mimir
 
+* [CHANGE] Alertmanager: Updated Alertmanager to latest main (`fa6a7e6`). This version of Alertmanager adds support for UTF-8 but is disabled in Mimir. It also deprecates Alertmanager APIv1, and replaces all APIv1 endpoints with a deprecation notice.
```

Comment:
[nit] A couple of nits:

1. In the CHANGELOG we only report information useful for the final user (e.g. no commit hashes or no reference to features not enabled in Mimir)
2. We add the PR number at the end of the CHANGELOG entry

So, something like this (but feel free to edit it):

```suggestion
* [CHANGE] Alertmanager: Alertmanager API v1 endpoints support have been removed. The API v1 endpoints will not work and will only report a deprecation notice. #7103
```

Reply:
But what's about `/v1/silence` and `/v1/silences`? Are they still supported? Maybe it's better if you list all the endpoints whose support has been removed.

Reply:
All APIv1 endpoints are deprecated upstream. I will make this clear in the CHANGELOG. Thanks for the feedback 👍